pom.xml

```diff
@@ -90,8 +90,13 @@
         <!-- TESTS -->
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
-            <version>1.10.19</version>
+            <artifactId>mockito-core</artifactId>
```

Comment:
We don't need mockito-all and mockito-core is managed by the parent project already.